src/edu/stanford/dlss/was/WasapiDownloader.java

```diff
@@ -1,40 +1,22 @@
 package edu.stanford.dlss.was;
 
 import java.io.IOException;
-import java.io.InputStream;
-import java.io.FileInputStream;
 import java.util.Properties;
 
+import edu.stanford.dlss.was.WasapiDownloaderSettings;
```

Comment:
It's not necessary to import classes from the same package, they are included by default.